liquid_node/commands.py

```diff
@@ -167,15 +170,18 @@ def deploy():
         'liquid/hoover/auth.django',
         'liquid/hoover/search.django',
         'liquid/authdemo/auth.django',
-        'liquid/nextcloud/nextcloud.admin',
-        'liquid/nextcloud/nextcloud.maria',
-        'liquid/dokuwiki/auth.django',
-        'liquid/nextcloud/auth.django',
-        'liquid/rocketchat/auth.django',
-        'liquid/ci/vmck.django',
-        'liquid/ci/drone.secret',
     ]
-    core_auth_apps = list(CORE_AUTH_APPS)
+    if scope != 'hoover':
```

Comment:
I don't think it's wise to special-case hoover, it should be just another app that we deploy.